this wasn't bumped for ffmpeg6, so bump it ourselves to not conflict with
ffmpeg 5. otherwise it can't be installed at the same time as ffmpeg 5 libs,
but we have multiple packages, even if the abi is the same.
--
--- a/libswresample/version.h
+++ b/libswresample/version.h
@@ -30,7 +30,7 @@
 
 #include "version_major.h"
 
-#define LIBSWRESAMPLE_VERSION_MINOR  12
+#define LIBSWRESAMPLE_VERSION_MINOR  0
 #define LIBSWRESAMPLE_VERSION_MICRO 100
 
 #define LIBSWRESAMPLE_VERSION_INT  AV_VERSION_INT(LIBSWRESAMPLE_VERSION_MAJOR, \
--- a/libswresample/version_major.h
+++ b/libswresample/version_major.h
@@ -26,6 +26,6 @@
  * Libswresample version macros
  */
 
-#define LIBSWRESAMPLE_VERSION_MAJOR   4
+#define LIBSWRESAMPLE_VERSION_MAJOR   5
 
 #endif /* SWRESAMPLE_VERSION_MAJOR_H */
